src/util/serviceSelector.ts

```diff
@@ -0,0 +1,106 @@
+import {  ProcessorRequest, RouterRequestData } from "../types/index";
+import { MiscService } from "../services/misc.service";
+import { INTEGRATION_SERVICE } from "../routes/utils/constants";
+import CDKV1ServiceDestination from "../services/destination/cdkV1Integration.destination.service";
+import CDKV2ServiceDestination from "../services/destination/cdkV2Integration.destination.service";
+import IntegrationServiceDestination from "../interfaces/IntegrationDestinationService";
+import NativeIntegrationServiceDestination from "../services/destination/nativentegration.destination.service";
+import IntegrationServiceSource from "../interfaces/IntegrationSourceService";
+import NativeIntegrationServiceSource from "../services/source/nativeIntegration.source.service";
```

Comment:
The naming is a bit odd. It should rather be `<implementation><module>Service` (ex: `CDKV1DestinationService`, `NativeSourceService` etc.).